lib/dams/controller_helper.rb

```diff
@@ -675,8 +675,12 @@ def audit( id = "unknown" )
     def referrer_controller( request )
       uri = URI(request.referrer || "")
       if uri.host == request.host
```

Comment:
Yes, I think that would be a way to look into this. It would be nice if the code and tests could use whatever the URL base is for the application. Given that we need to get this in the release today for Matt P to continue his work, I'll merge this as-is, and setup a ticket to see if we can remove these inconsistencies in the next Sprint. Sound good?